02.calendar/calendar2.rb

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env ruby
+
+require "optparse"
+require "Date"
```

Comment:
```suggestion
require "date"
```

Rubyで `require` する時はスネークケースを使いましょう。

https://designsupply-web.com/media/development/4052/